internal/powerplatform/services/environment/datasource_environments.go

```diff
@@ -58,8 +58,13 @@ func (d *EnvironmentsDataSource) Schema(_ context.Context, _ datasource.SchemaRe
 							Computed:            true,
 						},
 						"location": schema.StringAttribute{
-							Description:         "Location of the environment (europe, unitedstates etc.)",
-							MarkdownDescription: "Location of the environment (europe, unitedstates etc.)",
+							Description:         "Location of the environment (europe, unitedstates etc.). Can be queried using the `powerplatform_locations` data source",
+							MarkdownDescription: "Location of the environment (europe, unitedstates etc.). Can be queried using the `powerplatform_locations` data source",
+							Computed:            true,
+						},
+						"azure_region": schema.StringAttribute{
+							Description:         "Azure region of the environment (westeurope, eastus etc.). Can be queried using the `powerplatform_azure_regions` data source",
+							MarkdownDescription: "Azure region of the environment (westeurope, eastus etc.). Can be queried using the `powerplatform_azure_regions` data source",
```

Comment:
```suggestion
							Description:         "Azure region of the environment (westeurope, eastus etc.). Can be queried using the `powerplatform_azure_regions` data source. This property should only be set if absolutely necessary like when trying to create an environment in the same Azure region as Azure resources or Fabric capacity.  Changing this property after environment creation will result in a destroy and recreation of the environment (you can use the [`prevent_destroy` lifecycle metatdata](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#prevent_destroy) as an added safeguard to prevent accidental deletion of environments).",
							MarkdownDescription: "Azure region of the environment (westeurope, eastus etc.). Can be queried using the `powerplatform_azure_regions` data source. This property should only be set if absolutely necessary like when trying to create an environment in the same Azure region as Azure resources or Fabric capacity.  Changing this property after environment creation will result in a destroy and recreation of the environment (you can use the [`prevent_destroy` lifecycle metatdata](https://developer.hashicorp.com/terraform/language/meta-arguments/lifecycle#prevent_destroy) as an added safeguard to prevent accidental deletion of environments).",
```